nixos/modules/services/monitoring/prometheus/exporters/py-air-control.nix

```diff
@@ -0,0 +1,62 @@
+{ config, lib, pkgs, options }:
+
+with lib;
+
+let
+  cfg = config.services.prometheus.exporters.py-air-control;
+
+  py-air-control-exporter-env = pkgs.python3.withPackages (pyPkgs: [
+      pyPkgs.py-air-control-exporter
+  ]);
+
+  workingDir = "/var/lib/${cfg.stateDir}";
+
+in
+{
+  port = 9896;
```

Comment:
This port has been added to the [Prometheus' port allocations table](https://github.com/prometheus/prometheus/wiki/Default-port-allocations).